api/v1alpha1/plugin_manager.proto

```diff
@@ -92,5 +92,7 @@ message Wasm {
 
 message Inline {
     google.protobuf.Struct settings = 1;
+    bool directPatch = 2;
+    string fieldPatchTo = 3;
```

Comment:
this complicate matters

Reply:
It does get a bit complicated and there is a trade-off between flexibility and ease of use

Reply:
I think, we need to clarify function of plugin module: support all envoyfilter generation needs or focus on route and vhost configurations

Reply:
as described in README, plugin is a http plugin management that can only generate a limited type of envoyfilter. for EnvoyPlugin resources, plugin generates a patch that applies to either vhost or route, as appropriate.

Envoyfilter is designed to support to patch any field of the xds object (for merge scenarios), and users using envoyfilter are required to have the necessary knowledge of the xds api, as is the case with EvnoyPlugin. The current commit really allows users to freely modify vhost and route via EvnoyPlugin, just like envoyfilter does.

In fact, we are now focusing on routing and web hosting configuration.